.gitlab-ci.yml

```diff
@@ -142,7 +145,10 @@ include:
   - scripts/ci/gitlab/pipeline/test.yml
   # # build jobs
   - scripts/ci/gitlab/pipeline/build.yml
+  # short-benchmarks jobs
+  - scripts/ci/gitlab/pipeline/short-benchmarks.yml
   # # benchmarks jobs
+  # # TODO: can we remove this?-
   - scripts/ci/gitlab/pipeline/benchmarks.yml
```

Comment:
Looks like this one is automatically creating a MR to update the weights? I dont know if that is used.

Reply:
Correct, but it works only with `release-parachains-v*` branches

Reply:
@ggwpez yes, it is used for the releases, when the benchmark job is triggered from git lab. We used to do it manually so far

Reply:
@EgorPopelyaev @ggwpez @joepetrowski @chevdor 
so for the very next release to regenerate weights, do we want to use:
1. old way with manual gitlab job?
2. new `bot bench $ all` ?

Reply:
@bkontur lets's proceed with the new `bot` solution if it does a thing, but I would still keep the old pipeline yaml, as a plan B. At least for the next release

Reply:
@EgorPopelyaev ok, good point